src/javascript/app/base/header.js

```diff
@@ -561,12 +617,95 @@ const Header = (() => {
         Client.sendLogoutRequest();
     };
 
+    const populateWalletAccounts = () => {
+        if (!Client.isLoggedIn() || !Client.hasWalletsAccount()) return;
+        const account_list      = getElementById('wallet__switcher-accounts-list');
+        BinarySocket.wait('authorize', 'website_status', 'balance', 'landing_company', 'get_account_status').then(() => {
+            Client.getAllLoginids().forEach((loginid) => {
+                const is_wallet_account        = Client.isWalletsAccount(loginid);
+                if (!Client.get('is_disabled', loginid) && Client.get('token', loginid) && !is_wallet_account) {
+                    const is_real              = loginid.startsWith('CR');
+                    const currency             = Client.get('currency', loginid);
+                    const currencyName         = mapCurrencyName(currency);
+                    
+                    const getIcon              = (() => {
```

Comment:
Sorry, didn't get why do we need extra brackets here

Reply:
thats a typo. thanks for pointing it out